csrankings-r.csv

```diff
@@ -1323,7 +1323,7 @@ Rui Li 0002,Rochester Institute of Technology,https://www.rit.edu/directory/rxli
 Rui Mao 0001,Shenzhen University,https://www.sics.ac.cn/mao/szu/eng,MVbJ9wQAAAAJ
 Rui Ponte Costa,University of Bristol,https://research-information.bris.ac.uk/en/persons/rui-ponte-costa,otGgQKQAAAAJ
 Rui Prada,Universidade de Lisboa,http://gaips.inesc-id.pt/component/gaips/people/displayPerson/8/11,UvoCzwcAAAAJ
-Rui Tan,Nanyang Technological University,http://www.ntu.edu.sg/home/tanrui,co5MGPkAAAAJ
+Rui Tan,Nanyang Technological University,https://www3.ntu.edu.sg/home/tanrui,co5MGPkAAAAJ

```

Comment:
This is a disambiguation page and needs to be removed following the instructions.
https://dblp.org/pid/00/5179.html

> If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers ([dblp@dagstuhl.de](mailto:dblp@dagstuhl.de)) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

Reply:
I'm going to merge this PR but will move the faculty member to `old/other.csv`; please open an issue as directed above.